en/middleware/micrortps.md

```diff
@@ -231,31 +231,38 @@ Before building `px4_ros_com` you will first need to clone the PX4 Firmware repo
 Then install and setup both ROS2 and ROS environments on your development machine
 and separately clone the `px4_ros_com` repo for both the `master` and `ros1` branches (see [above for more information](#px4_ros_com)).
 
-> **Note** Only the master branch is needed for ROS2 (both are needed to target ROS). 
+> **Note** Only the master branch is needed for ROS2 (both are needed to target ROS).
 
 ### Installing ROS and ROS2 and respective dependencies
 
 In order to install ROS Melodic and ROS2 Bouncy on a Ubuntu 18.04 machine, follow the links below, respectively:
 
 1. [Install ROS Melodic](http://wiki.ros.org/melodic/Installation/Ubuntu)
 2. [Install ROS2 Bouncy](https://index.ros.org/doc/ros2/Linux-Install-Debians/)
-
-Install the following component to ensure that the package properly generates the IDL files:
+3. Install the following component to ensure that the package properly generates the IDL files (only required if one is using the ROS2 Bouncy release):
 
 ```sh
```

Comment:
@TSC21 Care needs to be made of indentation because you changed from a bullet to a numbered items. Things in bullets indent 2 spaces, while things under numbered items must indent 3 spaces. 

So basically everything under this list item must start on a line with three spaces - this code, the following note, the following line and the following code block (etc)

Reply:
Yeah sorry for that.